packages/core/metrics/core-metrics-collectors-server-internal/src/cgroup/cgroup.ts

```diff
@@ -0,0 +1,78 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import type { Logger } from '@kbn/logging';
+import type { MetricsCollector } from '@kbn/core-metrics-server';
+
+import * as v1 from './v1';
+import * as v2 from './v2';
```

Comment:
NIT: we usually avoid importing `*`, especially from within the same internal package. I would either use named imports, or even just rename the exported functions from the two files

```ts
import { gatherCgroupMetrics as gatherV1CgroupMetrics } from './v1';
import { gatherCgroupMetrics as gatherV2CgroupMetrics } from './v2';
```

or (by renaming from the import source directly)

```ts
import { gatherV1CgroupMetrics } from './v1';
import { gatherV2CgroupMetrics } from './v2';
```